fdbserver/RestoreApplier.actor.cpp

```diff
@@ -359,7 +361,7 @@ ACTOR Future<Void> applyToDB(Reference<RestoreApplierData> self, Database cx) {
 			    .detail("TxnStatus", "?")
 			    .detail("ApplierApplyToDB", self->id())
```

Comment:
I see other trace event uses "ApplierNode" for `self->id()`.
```suggestion
			    .detail("ApplierNode", self->id())

```

Reply:
This is on purpose. This `ApplierApplyToDB` is for the applying phase only.